lib/rucio/core/did.py

```diff
@@ -1861,19 +1862,29 @@ def list_parent_dids(
 
     :param scope:     The scope.
     :param name:      The name.
+    :param order_by:  List of parameters to order the query by.
```

Comment:
The docstring should list the allowed parameters or say explicitly what they're constrained by

Reply:
Added in latest force-push